Info.plist

```diff
@@ -113,9 +140,8 @@
 	<true/>
 	<key>tsInputMethodCharacterRepertoireKey</key>
 	<array>
-		<string>Hans</string>
```

Comment:
what does this change do?

Reply:
Only `Hant` and `Hans` are valid repertoire keys for `zh` (same for `yue` and several other Chinese languages, see [ISO 15924](https://en.wikipedia.org/wiki/ISO_15924)). So, delete the invalid `Han` repertoire.